Automattic/newspack-theme
# Add widget area above and below post content

### All Submissions:


### Changes proposed in this Pull Request:

This PR adds a widget area above and below the article content. 

Right now they are the same width as the content area (780px). We could also make them wider, so they are 1200px wide.

They also are directly above and below the content (and above the author bio and comments). I'm not sold on this so if there's a better spot to put them or width, please let me know!

Above content (is below featured image when one is set):

![image](https://user-images.githubusercontent.com/177561/64492309-86e19900-d227-11e9-89f0-31291f35ed0a.png)

Below Content:

![image](https://user-images.githubusercontent.com/177561/64492311-8c3ee380-d227-11e9-93ae-ef8a925a075a.png)

Closes #375 .

### How to test the changes in this Pull Request:

1. Apply the PR and run `npm run build`
2. Navigate to the Customizer > Widgets, and note you have two new areas -- "Article above content" and "Article below content"
3. Add a Custom HTML widget to each area and add the following code to test where they appear:

```
<div style="background: #ccc; height: 100px;">
	Test Content
</div>
```

4. View on a single post with the default template, and a single post with the 'Article feature' template.
5. Give some feedback on the location and appearance (width and placement) of the widget area.

### Other information:
